files/galaxy/tpv/tools.yml

```diff
@@ -106,6 +106,37 @@ tools:
     cores: 10
     mem: 32
     rules:
+        # Note: As of today, there is no `tool_version_eq` function in TPV.
+      - if: helpers.tool_version_gte(tool, '2.0.0+galaxy1') and helpers.tool_version_lt(tool, '2.0.0+galaxy2')
+        # meaning `helpers.tool_version_eq(tool, '2.0.0+galaxy1')`
+        # The version number of alphafold may not match the version number of the the tool in this case. The alphafold
+        # version number should in fact be newer than 2.1.0 and older than 2.1.1. Check the links below to verify this
+        # claim:
+        # - https://github.com/usegalaxy-au/tools-au/blob/fae57866fda74c85d405a2db03a82ebfdaed6070/tools/alphafold/docker/Dockerfile#L7
+        # - https://github.com/usegalaxy-au/tools-au/tree/fae57866fda74c85d405a2db03a82ebfdaed6070/tools/alphafold/docker
+        # - https://github.com/usegalaxy-au/tools-au/commit/aa7c146a02df64fcaa8ef03a89a76012227f35d6
+        # - https://github.com/deepmind/alphafold/blob/be37a41d6f83e4145bd4912cbe8bf6a24af80c29/setup.py#L21
+        # However neither, 2.1.0 nor 2.1.1 are still supposed to be able to use the GPU during the relaxation step, yet
+        # there are CUDA errors in the tool tests when no GPU is available. Verify the GPU use claim below:
+        # - https://github.com/deepmind/alphafold/blob/be37a41d6f83e4145bd4912cbe8bf6a24af80c29/alphafold/relax/amber_minimize.py#L93
+        # There seems indeed to have been a mishap when packaging this version of the tool. See the link below.
+        # - https://github.com/usegalaxy-au/tools-au/commit/06dd35df4064c0c3c1272957e46c0df59d24c7fe
+        # Regardless of how things came to be this way, this version of the tool needs a GPU, and the
+        # requirement cannot be disabled) because `ALPHAFOLD_USE_GPU` is not declared in alphafold.xml. Thus we set
+        # gpus to one.
+        gpus: 1
+      - if: helpers.tool_version_gte(tool, '2.0.0+galaxy2') and helpers.tool_version_lt(tool, '2.0.0+galaxy3')
+        # meaning `helpers.tool_version_eq(tool, '2.0.0+galaxy2')`
```

Comment:
Version '2.0.0+galaxy2' is not installed on usegalaxy.eu but exists. If we ever install it this will prevent the problem.